packages/zowe-explorer-ftp-extension/src/abstractFtpApi.ts

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
A better file name like `ZoweExplorerAbstractFtpApi.ts`.

Reply:
Thanks Qi! I updated the name to ZoweExplorerAbstractFtpApi.ts.